test/array.find.js

```diff
@@ -15,27 +16,64 @@ metatests.test('find with error', test => {
       }
     });
 
-  metasync.find(data, predicate, err => {
+  metasync.find(arr, predicate, err => {
     test.type(err, 'Error', 'err must be an instance of Error');
     test.strictSame(err.message, expectedErrorMessage);
     test.end();
   });
 });
 
-metatests.test('find', test => {
-  const data = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16];
+metatests.test('find / array', test => {
   const expected = 15;
   const predicate = (item, callback) =>
     process.nextTick(() => callback(null, item % 3 === 0 && item % 5 === 0));
 
+  metasync.find(arr, predicate, (err, result) => {
+    test.error(err, 'must not return an error');
+    test.strictSame(result, expected, `result should be: ${expected}`);
+    test.end();
+  });
+});
+
+metatests.test('find / set', test => {
+  const data = new Set([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16]);
```

Comment:
Why you don't use `arr`?

Reply:
Because I am testing `Set` collection. I can remove this test exactly for `Set`.